ddtrace/tracer/abandonedspans_test.go

```diff
@@ -44,15 +44,16 @@ func assertProcessedSpans(assert *assert.Assertions, t *tracer, startedSpans, fi
 		return atomic.LoadUint32(&d.addedSpans) >= uint32(startedSpans) &&
 			atomic.LoadUint32(&d.removedSpans) >= uint32(finishedSpans)
 	}
-	assert.Eventually(cond, 500*time.Millisecond, 50*time.Millisecond)
+	assert.Eventually(cond, 1*time.Second, 75*time.Millisecond)
```

Comment:
it's okay to put longer timeouts if we're close ti 1 second already.

Reply:
Usually it takes quite less, but I jumped up to 1 second to avoid flakiness. Anyhow, now it should be completely solid 😄